screengrab/screengrab-lib/src/main/java/tools.fastlane.screengrab/locale/LocaleUtil.java

```diff
@@ -25,8 +25,12 @@ public static void changeDeviceLocaleTo(Locale locale) {
             methodGetDefault.setAccessible(true);
             Object activityManagerNative = methodGetDefault.invoke(amnClass);
 
-            Class iam = Class.forName(activityManagerNative.getClass().toString());
-            Method methodGetConfiguration = iam.getMethod("getConfiguration");
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
22 works now but 24 fails with:

```
[23:42:59]: ▸ Caused by: java.lang.ClassNotFoundException: Invalid name: class android.app.ActivityManagerProxy
[23:42:59]: ▸ at java.lang.Class.classForName(Native Method)
[23:42:59]: ▸ at java.lang.Class.forName(Class.java:400)
[23:42:59]: ▸ at java.lang.Class.forName(Class.java:326)
[23:42:59]: ▸ at tools.fastlane.screengrab.locale.LocaleUtil.changeDeviceLocaleTo(LocaleUtil.java:30)
```

Should not the check be `Build.VERSION.SDK_INT >= Build.VERSION_CODES.O` since nothing has changed on M / N as screengrab worked on those perfectly fine?